### PR TITLE
[Storybook] Introduce Concept Theme page

### PR DESCRIPTION
**Pull request by JoshuaLai**
# What
Storybook v8 migration 
image
image
image

# Why
Storybook 8 migration

### CODE REVIEW

**Comment by prabhjot-msft** [comment]
Should it have index.stories and have export stories and use canvas to render?

File: packages/storybook8/stories/Concepts/Theming/snippets/DarkControlBar.snippet.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```
Comment (jimchou-dev) [inline packages/storybook8/stories/Concepts/Theming/snippets/DarkControlBar.snippet.tsx:2]: do snippet need this header comment?

File: packages/storybook8/stories/Concepts/Theming/Docs.mdx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+<DefaultTheme />
+
+To theme this ControlBar, import `FluentThemeProvider` from `@azure/communication-react` and wrap it around the `ControlBar`.
+Then import `darkTheme` from `@azure/communication-react` and assign it to the `fluentTheme` property of `FluentThemeProvider`.
+`darkTheme` is a [Theme](https://aka.ms/fluentui#/controls/web/references/theme) object from Fluent UI.
+
+<Source code={DarkControlBarText} />
+<DarkControlBar />
+
+## FluentThemeProvider Props
```
Comment (jimchou-dev) [inline packages/storybook8/stories/Concepts/Theming/Docs.mdx:32]: I assume we're still trying to figure out props?

Comment (JoshuaLai) [reply packages/storybook8/stories/Concepts/Theming/Docs.mdx:32]: Yes this is an ongoing issue we need to figure out

File: packages/storybook8/stories/Concepts/Theming/snippets/DarkControlBar.snippet.tsx (hunk 1 above)
Comment (JoshuaLai) [reply packages/storybook8/stories/Concepts/Theming/snippets/DarkControlBar.snippet.tsx]: Good catch, removed

File: packages/storybook8/stories/Concepts/Theming/Docs.mdx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+<Source code={DefaultThemeSnippetText} />
+<DefaultTheme />
+
+To theme this ControlBar, import `FluentThemeProvider` from `@azure/communication-react` and wrap it around the `ControlBar`.
+Then import `darkTheme` from `@azure/communication-react` and assign it to the `fluentTheme` property of `FluentThemeProvider`.
+`darkTheme` is a [Theme](https://aka.ms/fluentui#/controls/web/references/theme) object from Fluent UI.
+
+<Source code={DarkControlBarText} />
+<DarkControlBar />
```
[the remark this answers is not in the document]
Comment (JoshuaLai) [reply packages/storybook8/stories/Concepts/Theming/Docs.mdx:30]: I want to merge it in as is while we discuss how to address these tiny little UI inconsistencies and see if its acceptable or if they have to be exactly the same

**Review by jimchou-dev — APPROVED** [review]

**Review by prabhjot-msft — APPROVED** [review]

**Pull request merged**